hooks/useFilteredUtmParams.ts

```diff
@@ -0,0 +1,30 @@
+// TODO: GIVCAMP-164
```

Comment:
These utm related functions will need to be redone. Added ticket and a TODO here so it's not just empty promises 😂